### PR TITLE
Swift 3

### PR DESCRIPTION
**Pull request by evandcoleman**
This PR updates this repo to Swift 3.

### CODE REVIEW

File: KDEDateLabel.swift [hunk 1]
```diff
@@ -11,23 +11,23 @@ import UIKit
 // MARK: - KDEWeakReferencer
 ////////////////////////////////////////////////////////////////////////////////
 
-private class KDEWeakReferencer<T: NSObject>: NSObject {
+internal class KDEWeakReferencer<T: NSObject>: NSObject {
```
Comment (delannoyk) [inline KDEDateLabel.swift]: Could you use `fileprivate` here?

File: KDEDateLabel.swift [hunk 2]
```diff
@@ -38,43 +38,43 @@ private class KDEWeakReferencer<T: NSObject>: NSObject {
 // MARK: - KDEDateLabelsHolder
 ////////////////////////////////////////////////////////////////////////////////
 
-class KDEDateLabelsHolder: NSObject {
-    private var dateLabels = [KDEWeakReferencer<KDEDateLabel>]()
-    private var timer: NSTimer?
+internal class KDEDateLabelsHolder: NSObject {
```
Comment (delannoyk) [inline KDEDateLabel.swift]: Could you use `fileprivate` here?

File: KDEDateLabel.swift [hunk 3]
```diff
@@ -38,43 +38,43 @@ private class KDEWeakReferencer<T: NSObject>: NSObject {
...
+    var dateLabels = [KDEWeakReferencer<KDEDateLabel>]()
+    var timer: Timer?
```
Comment (delannoyk) [inline KDEDateLabel.swift]: ^^ I think these should remain `private`

File: KDEDateLabel.swift [hunk 4]
```diff
@@ -119,39 +119,39 @@ public class KDEDateLabel: UILabel {
...
 
-    public var dateFormatAttributedTextBlock: ((date: NSDate) -> NSAttributedString)? {
+    open var dateFormatAttributedTextBlock: ((_ date: Date) -> NSAttributedString)? {
         didSet {
             self.updateText()
         }
     }
 
-    private func updateText() {
+    internal func updateText() {
```
Comment (delannoyk) [inline KDEDateLabel.swift]: Could this remain `private` or `fileprivate`?

**Review by delannoyk — CHANGES_REQUESTED** [review]
Hey! Thanks for updating this! Left some quick comments regarding minor access control issues.

**Comment by evandcoleman** [comment]
@delannoyk Thanks for taking a look! Just push a commit with those changes.

**Comment by delannoyk** [comment]
@edc1591 Cool, thanks for this!

**Pull request merged**